docs/id-lifecycle-management/support-systems/partner-management-services/functional-overview/onboard-credential-partner.md

```
@@ -1,80 +1,67 @@
-    13. Organization Name
-    14. Address, Phone number
-    15. Email, Username and password
+1. The Credential Partner can register themselves on MOSIP PMS portal by clicking **Register** on the Login Page, a form comes up.
+2. Enter the Authentication Partner details:
+    1. Partner type (Authentication Partner)
+    2. First and Last name
+    3. Organization Name
+    4. Address, Phone number
+    5. Email, Username and password
 
 ![](../../../../.gitbook/assets/pms-b5-ocp-image1.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)